test/dag-node-test.js

```diff
@@ -4,8 +4,10 @@
 const chai = require('chai')
 const { Buffer } = require('buffer')
 const dirtyChai = require('dirty-chai')
+const chaiSubset = require('chai-subset')
```

Comment:
You can require chai from `aegir/utils/chai` and it has `dirty-chai` and `chai-subset` preconfigured. It also avoids nasty weird bugs related to the order of `chai.use` invocations [across the entire codebase](https://github.com/chaijs/chai/issues/1298).